<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectWindow</class>
 <widget class="QDialog" name="SubjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #A4A8A6;</string>
  </property>
  <widget class="QScrollArea" name="scroll_qa">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>631</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF5B2;
border: 1px solid;
border-color: #FFCF38;
font-size: 15px;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>629</width>
      <height>539</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_subject">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-weight: bold;
font-size: 18px;
border: 1px solid;
background-color: #06A310;</string>
   </property>
   <property name="text">
    <string>SUBJECT : &lt;name&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_delete_col">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0048FF;
font-size: 12px;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_subj_col">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0048FF;
font-size: 12px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>QUESTIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_instructions">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 14px;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>Click on the questions to reveal the anwer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info_icon">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/info icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>&lt;&lt; BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>Add Question</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
